stories/features/rewards/other.tsx

```diff
@@ -209,3 +214,13 @@ storiesOf('Feature Components/Rewards/Other', module)
         />
       )
     })
+    .add('Toggle Tips', () => {
+      return (
+        <ToggleTips
+          id={'toggle-tips'}
+          onToggleTips={dummyToggle}
```

Comment:
Please add state to the story, so that toggle will actually change when you click on it